MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -32,6 +30,10 @@ namespace MediaBrowser.Api.Playback
     /// </summary>
     public abstract class BaseStreamingService : BaseApiService
     {
+        protected static readonly CultureInfo UsCulture = CultureInfo.ReadOnly(new CultureInfo("en-US"));
```

Comment:
UsCulture variable is only used for int and double parsing and one formatting call. We should use `CultureInfo.InvariantCulture` I believe.